### DISCUSSION TITLE
IOS maps, download maps are outdated

### DISCUSSION THREAD

**Discussion by ismailiconium** [organicmaps/organicmaps#8173] [Q&A]
I installed the app from appstore for ios. The whole maps for Turkiye (=Turkey) is 635,3 MB, i have maps.me installed on the same device with whole maps for Turkiye 1,16 GB. what is the cause of difference, are the linked to different repo's for downloading the maps, or does maps.me add additional information to there maps. They should use the same source for download, assuming the download from openstreetmaps..

**Comment by ismailiconium** [comment]
[[https://github.com/orgs/organicmaps/discussions/8174](url), 
Android & iOS maps not synced #8174
may be the same issue/cause

**Reply by biodranik** [reply]
No, that is not an issue at all.

**Reply by ismailiconium** [reply]
What is then the issue, why is there difference in size between the turkey organic maps versus turkey maps.me maps? Can we solve this from the user/us?

**Reply by biodranik** [reply]
I do not know what has been added/included in other apps/maps. Maybe someone wants to investigate and compare.

**Reply by ismailiconium** [reply]
do you think maps.me added extra data to original openstreetmaps? you are right, if i zoom in at both maps, then maps.me has detailed information with building names, firm names etc... this can be the cause of the file size. both maps doest have street names....

thanks

**Reply by biodranik** [reply]
I doubt it. Organic Maps has more data from OSM.

**Reply by ismailiconium** [reply]
thread getting more interesting... then the question remains, what is the cause of different in sizes between maps?

**Reply by biodranik** [reply]
You can try to check it using generator_tool. It has some flags to print map stats.

**Reply by pastk** [reply]
I had a quite old MM version (maybe ~1yr old) and I've just updated.
Map sizes grew indeed, e.g. West Bulgaria was 90Mb, now its 112Mb (in OM its 101Mb atm).
India Uttarakhand is 53Mb in MM and 34Mb in OM (was almost the same before). Isolines density is the same.

**Comment by map-per** [comment]
> IOS maps, download maps are outdated

If you use the latest version and did all map updates the data in Organic Maps (OM) is definitely not outdated, but form May 11th 2024. You can check by clicking on the OM logo in the tool bar. The About OM page lists how old the map data is.

> They should use the same source for download, assuming the download from openstreetmaps..

The maps used in OM are not directly downloaded from OpenStreetMap (OSM). The maintainers fetch the OSM data about once a month, do some pre-processing and compress the data. The OM app then downloads the data from the OM Servers. The good compression of OM is the reason why map sizes in OM are smaller than in most other applications. But this doesn't explain the difference to Maps.me, as Maps.me probably (it's closed source, so hard to check) uses a very similar compression.
